src/main/resources/liquibase/harness/compatibility/foundational/expectedResultSet/mssql/azure/createTable.json

```diff
@@ -4,7 +4,7 @@
       "EXECTYPE":"EXECUTED",
       "COMMENTS":"",
       "AUTHOR":"liquibase",
-      "MD5SUM":"8:22fecc43185615efdc0426bf8b4c2a1b",
+      "MD5SUM":".*",
```

Comment:
Is this one expected?

Reply:
short answer - yes.
this is for init script. We may need to change init scripts and that will cause the test to fail. It's not this test job to check md5sum for init script, we only check md5sum input files.